File: pgm/ABC_C#/ABC_141_A.cs
using System;
// using System.Collections.Generic;
// using System.Linq;
// using System.Text;

class Program
{
    static void Main()
    {
        string S = Console.ReadLine();
        string[] W = { "Sunny", "Cloudy", "Rainy" };
        Console.WriteLine(W[(Array.IndexOf(W, S) + 1) % 3]);
        return;
    }
}

File: pgm/ABC_Visual_Basic/ABC_141_A.vb
' Imports System.Collections.Generic
' Imports System.Linq
' Imports System.Text
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim S = Console.ReadLine()
        Dim W = {"Sunny", "Cloudy", "Rainy"}
        Console.WriteLine(W((Array.IndexOf(W, S) + 1) Mod 3))
        Return
    End Sub
End Class